test/observers.spec.js

```diff
@@ -0,0 +1,183 @@
+'use strict';
+import observers from '../src/apps/observers';
+import findFreePort from 'find-free-port';
+import nock from 'nock';
+import { Server as mockSocketServer, WebSocket } from 'mock-socket';
+import EventEmitter from 'eventemitter3';
+import { expect } from 'chai';
+import { NEW_BUNDLE } from '../src/apps/actions';
+
+const API_PREFIX = '/api/v0';
+
+class FakeStore {
```

Comment:
It may be worthwhile to describe how the API of this store diverges from a real store.

For example,

- you can set state directly into it whereas you can't with a real store
- you can only register one function via `subscribe()`
- `dispatch()` doesn't trigger `subscribe()`'s callback